xproc/src/main/xml/specification.xml

```diff
@@ -2812,31 +2795,52 @@ they are <firstterm>visible</firstterm> to each other.
       <para><error code="S0036">All the step types in a pipeline or library <rfc2119>must</rfc2119>
           have unique names: it is a <glossterm>static error</glossterm> if any step type name is
           built-in and/or declared or defined more than once in the same scope.</error></para>
-
-<para>The scope of the names of the steps themselves is determined by
+</section>
+      
+<section>
+  <title>Scoping of step names</title>
+<para>The scope of the names of the steps (the values of the step's <code>name</code> attributes) is determined by
```

Comment:
Please use typographic quotes (U+2019): step’s

Reply:
I changed the one you pointed out, but a quick search showed me a 's is used in 55 places. I suggest we do these kinds of finalizing editorial changes in somewhere near the end.